src/extension/background-script/actions/ln/auth.ts

```diff
@@ -0,0 +1,16 @@
+import { MessageLnurlAuth } from "~/types";
```

Comment:
the `ln` folder is used for actions that call LN functions on the connector. 
so this does not fit in here. we have a lnurl folder, don't we?

Reply:
Yes, will move. We talked about it but I wasn't sure.

Reply:
Have a look again